src/mock/handlers/auth.ts

```diff
@@ -22,4 +23,44 @@ export default [
         }
         return res(ctx.cookie('jwt', 'not a real jwt'), ctx.status(200));
     }),
+
+    rest.post('/api/users/consume-password-reset-token', (req, res, ctx) => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const { token, form } = req.body as {
+            token: string;
+            form: AuthTypes.ForgotPassForm;
+        };
+        if (form.password === 'fail') {
+            return res(ctx.status(400));
+        }
+        return res(ctx.status(200, 'Password Reset'));
+    }),
+
+    rest.post('/api/users/request-password-reset', (req, res, ctx) => {
+        const { form } = req.body as {
+            form: AuthTypes.ForgotPassRequestForm;
+        };
+        if (form.email === 'fail') {
+            res(ctx.status(400));
+        }
```

Comment:
Does this work?  It might not ever send this because `fail` isn't a valid email and I think our component or api request requires that the input be a valid email before the request ever gets sent.

Reply:
Yes, you are right. I fixed it.